src/sass/_common/header.scss

```diff
@@ -223,7 +223,7 @@ body .header {
                 text-decoration: none;
                 letter-spacing: 0;
                 text-transform: capitalize;
-                padding: 0 8px;
+                padding: 0 0.8rem;
```

Comment:
does rem in binary-static project equal to 10px?

Reply:
Actually no, it's all over the place. I'll change back to 8px. I think this was before i found out it's all over the place.